mlir/lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -1193,10 +1192,18 @@ LogicalResult CppEmitter::emitAttribute(Location loc, Attribute attr) {
 
   // Print floating point attributes.
   if (auto fAttr = dyn_cast<FloatAttr>(attr)) {
+    if (!fAttr.getType().isF32() && !fAttr.getType().isF64()) {
```

Comment:
Maybe this is more readable (nit).
```suggestion
    if (!isa<Float32Type, Float64Type>(fAttr.getType()) {
```